staff/pau-julian/bash/README.md

```diff
@@ -0,0 +1,110 @@
+# COMMANDS IN BASH
+
+Command in line of typical commands in Bash.
```

Comment:
proposal: change to "Typical inline commands in Bash"

Reply:
done